lorentz_force/lorentz_unit_tests.py

```diff
@@ -0,0 +1,49 @@
+"""
+Module to test lorentz_force.py
+"""
+
+import math
+from lorentz_force import lorentz_force
+
+def test_lorentz_force_at_90_degrees():
```

Comment:
Potential for floating-point precision issues in test_lorentz_force_at_0_degrees()